server/protocols/jmap-rfc-8621/src/main/scala/org/apache/james/jmap/http/SessionSupplier.scala

```diff
@@ -0,0 +1,76 @@
+/** **************************************************************
+ * Licensed to the Apache Software Foundation (ASF) under one   *
+ * or more contributor license agreements.  See the NOTICE file *
+ * distributed with this work for additional information        *
+ * regarding copyright ownership.  The ASF licenses this file   *
+ * to you under the Apache License, Version 2.0 (the            *
+ * "License"); you may not use this file except in compliance   *
+ * with the License.  You may obtain a copy of the License at   *
+ * *
+ * http://www.apache.org/licenses/LICENSE-2.0                 *
+ * *
+ * Unless required by applicable law or agreed to in writing,   *
+ * software distributed under the License is distributed on an  *
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       *
+ * KIND, either express or implied.  See the License for the    *
+ * specific language governing permissions and limitations      *
+ * under the License.                                           *
+ * ***************************************************************/
+
+package org.apache.james.jmap.http
+
+import org.apache.james.core.Username
+import org.apache.james.jmap.model._
+
+object SessionSupplier {
+  private val MAX_SIZE_UPLOAD = UnsignedInt(50_000_000)
+  private val MAX_CONCURRENT_UPLOAD = UnsignedInt(8)
+  private val MAX_SIZE_REQUEST = UnsignedInt(10_000_000)
+  private val MAX_CONCURRENT_REQUESTS = UnsignedInt(8)
+  private val MAX_CALLS_IN_REQUEST = UnsignedInt(32)
+  private val MAX_OBJECTS_IN_GET = UnsignedInt(256)
+  private val MAX_OBJECTS_IN_SET = UnsignedInt(128)
+  private val URL = "http://this-url-is-hardcoded.org"
+  private val STATE = "thisIsHardCodedState"
+  private val ACCOUNT_ID = Id("thisIsHardCodedId")
+
+  private val CORE_CAPABILITY = CoreCapability(
+    maxSizeUpload = MAX_SIZE_UPLOAD, maxConcurrentUpload = MAX_CONCURRENT_UPLOAD,
+    maxSizeRequest = MAX_SIZE_REQUEST, maxConcurrentRequests = MAX_CONCURRENT_REQUESTS,
+    maxCallsInRequest = MAX_CALLS_IN_REQUEST, maxObjectsInGet = MAX_OBJECTS_IN_GET, maxObjectsInSet = MAX_OBJECTS_IN_SET,
+    collationAlgorithms = List())
+  private val MAIL_CAPABILITY = MailCapability(
+    maxMailboxDepth = None,
+    maxMailboxesPerEmail = None,
+    maxSizeMailboxName = UnsignedInt(255),
+    maxSizeAttachmentsPerEmail = UnsignedInt(25_000_000),
+    emailQuerySortOptions = List(),
+    mayCreateTopLevelMailbox = true)
+  private val CAPABILITIES = Map(
+    CapabilityIdentifier.JMAP_CORE -> CORE_CAPABILITY,
+    CapabilityIdentifier.JMAP_MAIL -> MAIL_CAPABILITY)
+
+  def generate(username: Username): Session = {
+    Session(
+      capabilities = CAPABILITIES,
+      accounts = accounts(username),
+      primaryAccounts = primaryAccounts(username),
+      username = username,
+      apiUrl = URL,
```

Comment:
Please use the same hardcoded bases for URL but ensure the endpoints are different